test/extended/builds/pipeline.go

```diff
@@ -200,7 +201,7 @@ var _ = g.Describe("[builds][Slow] openshift pipeline build", func() {
 			br.AssertSuccess()
 
 			g.By("confirm all the log annotations are there")
-			_, err = jenkins.ProcessLogURLAnnotations(oc, br)
+			err = jenkins.ProcessLogURLAnnotations(oc, br, !useSnapshotImage)
```

Comment:
I figured out the v2 image hiccup and am crafting PRs to switch to v2 for the PR testers ... closed this PR.